### DISCUSSION TITLE
I need help in creating a Dockerfile for installing the nextjs-starter on my coolify server

### DISCUSSION THREAD

**Discussion by Abz2604** [medusajs/medusa#7172] [Q&A]
This is for learning purposes, as I'm new to medusa

I have created a medusa storefront that runs on local using the npx create-medusa-project command with --next-js-starter

I was able to run this storefront on local

Now the next step would be to create a Dockerfile that helps me upload this project to a coolify server

Would greatly appreciate any help with this

**Comment by AKRking** [comment]
Have u done it successfully?
